src/components/screens/content-editor-screen.tsx

```diff
@@ -994,9 +1034,14 @@ class ContentEditorScreen extends React.Component<Props, State> {
     );
 
     const pages: ExhibitionPage[] = devicePages.flat();
-    const selectedDevice = devices[0];
     const selectedContentVersion = contentVersions[0];
-    const selectedPage = pages.find(page => page.deviceId === selectedDevice.id && page.contentVersionId === selectedContentVersion.id);
+    const selectedDevice = devices[0];
+    const previewDevices = this.getPreviewDevices(selectedContentVersion, devices, pages);
```

Comment:
previewDevices can be a bit misleading because it actually contains object containing device and page. All previewDevices variables and methods should be renamed to previewDeviceDatas